doc/specs/stdlib_experimental_io.md

```diff
@@ -6,14 +6,14 @@ title: experimental_IO
 
 [TOC]
 
-## `loadtxt` - load a 2D array from a text file
+## `[[stdlib_experimental_io(module):loadtxt(interface)]]` - load a 2D array from a text file
```

Comment:
I know I'm to blame for this, since I started putting links in the headings, but, JUST as an FYI and point for further discussion, [this breaks the TOC markdown extension](https://stdlib.fortran-lang.org/page/specs/stdlib_experimental_error.html) and in the TOC there are no longer links to the heading, only the link from within the heading to the linked FORD documentation.

IMO, this is a bug in the python markdown extra package (or perhaps FORD, but this is less likely) but I just wanted to point out this issue.

Reply:
I removed them. Thanks for the explanation.
I think it would be good to have a link from the spec to the code. Weprobalbly could add something like:

See the code for loadtxt here

after the title.

Reply:
I like this idea!